Cargo.toml

```diff
@@ -1,6 +1,6 @@
 [package]
 name = "tycho-simulation"
-version = "0.161.0"
+version = "0.160.1"
```

Comment:
why did this change here?

Reply:
i have changed this back like 5 times, i don't know why it keeps reverting to 1.60.1 -_-